nilearn/image/image.py

```diff
@@ -835,6 +836,10 @@ def clean_img(imgs, sessions=None, detrend=True, standardize=True,
     t_r: float, optional
         Repetition time, in second (sampling period).
 
+    ensure_finite: bool, optional
+        If True, the infinite values (NANs and infs) found in the images
+        will be replaced by zeros.
```

Comment:
>Although we agree that NaN = 0 in this case, I don't know what we should do with infinities... Maybe raise an error?

Rather than raising an error. We do what we have done for NaN case. May be convert them equivalently to zeros or less risk value. What do you say ? Can you say we occur infinity errors too more often ?